pkg/collector/corechecks/systemd/systemd_test.go

```diff
@@ -966,12 +966,7 @@ func TestGetPropertyBool(t *testing.T) {
 type mockAutoConfig struct{}
 
 func (*mockAutoConfig) GetLoadedConfigs() map[string]integration.Config {
-	ret := make(map[string]integration.Config)
-	ret["systemd_digest"] = integration.Config{
-		Name:     "systemd",
-		Provider: "provider1",
-	}
-	return ret
+	return make(map[string]integration.Config)
```

Comment:
```
	ret["systemd_digest"] = integration.Config{
		Name:     "systemd",
		Provider: "provider1",
	}
```

^ this config will create an entry with `config.hash -> systemd`. Since now the hash of the real instances are like `config.hash -> systemd:264b5c8c3f3978b1`, this will lead to the creation of two instances like those below in TestGetVersion test. Defining `systemd_digest` seems not necessary.

Two instances:

![image](https://user-images.githubusercontent.com/49917914/101488423-e1ecc900-395f-11eb-8a62-d77e1b47410a.png)